tests/components/profile_popover.test.jsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import ProfilePopover from 'components/profile_popover/profile_popover';
+
+describe('components/ProfilePopover', () => {
+    const baseProps = {
+        user: {name: 'name'},
+        src: 'src',
+        enableWebrtc: false,
```

Comment:
This prop is no longer used. I think I need to revisit and improve this test. I wanted to add the general snapshot case to get the test file established, but it may need to be tweaked. For instance there are some `@undefined`s in the snapshot that could probably be asserted better with values.

Reply:
Right, `enableWebrtc` is no longer used as we've dropped support to WebRTC recently.
For the `@undefined`, it's caused by the user prop with missing username field.

Reply:
Webrtc references removed. Username added to the `profile_popover` test.